index.js

```diff
@@ -442,7 +439,7 @@ program
           if (error) {
             logError(error, ERROR.CREATE);
           } else {
-            let scriptId = res.scriptId;
+            let scriptId = res.data.scriptId;
```

Comment:
Please destructure `data` here and elsewhere in the function callback above.

```js
- script.projects.create({ title, parentId }, {}, (error, res) => {
+ script.projects.create({ title, parentId }, {}, (error, {data}) => {
```

Reply:
done